jobs/scripts/parallel-tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+sed -i 's/    mr.developer/    mr.developer\ngit-clone-depth = 10/' core.cfg
+
+pip install -r requirements.txt
+buildout -c py3.cfg
+
+xvfb-run -a --server-args='-screen 0 1920x1200x24' bin/test -j 6 --xml -vvv
```

Comment:
We'll also need to export the `ROBOT_BROWSER` variable to `chrome` and add `chromedriver` to `$PATH`.